modmail/bot.py

```diff
@@ -27,6 +27,16 @@
     emojis_and_stickers=True,
 )
 
+TORTOISE_ORM = {
```

Comment:
perhaps this can be exported to a seperate file? This *possibly* could be in the configuration file, since it is a configuration value.

I don't really like this in bot.py

Reply:
It is like a constant, not really a configuration file as it is not modifiable.. so not sure. I can insert it directly in `Tortoise.init` if you want but I wouldn't be a fan of that.

Reply:
I think its a constant which is useful outside of this file, but this doesn't seem like a great place that other files should import that from, which is why I think it should be in a different file.

Reply:
Useful outside of this file, could you give some examples?

Reply:
Plugins may want to be able to see the connections and apps.

Reply:
Plugins would be using `self.bot.db`, they would never be interacting with variables, at least I don't see the need to.